public/js/reports/c3Charts.service.js

```diff
@@ -126,6 +125,11 @@ angular.module('app.reports').service('c3Charts', function (notify, gettextCatal
 
         if (!chart.height) { chart.height = 300; }
 
+        const dataFormatted = {};
+        report.properties.ykeys.forEach(d => {
+            dataFormatted[d.id] = d.format;
+        });
```

Comment:
I find the name confusing. In a variable named `dataFormatted` I expect to find "data that is formatted". Here we have only the format, not the data itself. `dataFormat` would be more appropriate I think. Or maybe `columnFormat` (as it is indexed by column id) ?

Reply:
I agree with you, that's little confusing. I changed names in my last commit, let me know if it's not better